### PR TITLE
UI alert if the network call fails 

### PR DESCRIPTION
**Pull request by steven-hill**
This change completes the search experience if the network call fails. It lets the user know there was an error and they should try again.

This includes a UIAlert and it returns the user to the SearchVC to try another search.

### CODE REVIEW

**Comment by OlivAlex** [comment]
- Lets go for a shorter title, we can remove things like "I added".

File: MovieList/MovieList/MovieList/MovieListCoordinator.swift [hunk 1]
```diff
@@ -32,7 +32,7 @@ extension MovieListCoordinator: MovieListVCDelegate {
                     self.movieController.updateUI()
 
                 case .failure(_):
-                    // TODO: pass alert back to MovieListVC.
+                self.movieController.networkCallAlert()
                     break
```
Comment (OlivAlex) [inline MovieList/MovieList/MovieList/MovieListCoordinator.swift]: You don't need the `break` in the switch and I _think_ indentation is wrong. Ctrl + I on the whole switch should fix it.

File: MovieList/MovieList/MovieList/MovieListCoordinator.swift [hunk 2]
```diff
@@ -57,6 +57,10 @@ extension MovieListCoordinator: MovieListVCDelegate {
         movieController.navigationController?.pushViewController(movieDetailVC, animated: true)
     }
     
+    func returnToSearchVC() {
+        movieController.navigationController?.popToRootViewController(animated: true)
```
Comment (OlivAlex) [inline MovieList/MovieList/MovieList/MovieListCoordinator.swift:60]: Whitespace here.

**Review by OlivAlex — COMMENTED** [review]
LGTM!

File: MovieList/MovieList/MovieList/MovieListCoordinator.swift (hunk 2 above)
Comment (steven-hill) [reply MovieList/MovieList/MovieList/MovieListCoordinator.swift:60]: I've checked the project and there's no whitespace in returnToSearchVC(). Maybe a GitHub thing?

File: MovieList/MovieList/MovieList/MovieListCoordinator.swift (hunk 1 above)
Comment (steven-hill) [reply MovieList/MovieList/MovieList/MovieListCoordinator.swift]: Removed the 'break' and re-indented the file.

**Pull request merged**